src/lib/storage/materialize.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <optional>
+
+#include "resolve_type.hpp"
+#include "storage/base_column.hpp"
+#include "storage/create_iterable_from_column.hpp"
+
+namespace opossum {
+
+/**
+ * Materialization convenience functions. Can't be put into base_column.hpp because that would lead to circular
+ * includes.
+ *
+ * Use like:
+ *
+ * pmr_vector<std::optional<T>> values_and_nulls;
```

Comment:
`std::pair<bool, T>` ;)

Also, I would indent the example by two spaces. This is how I did it everywhere else.

Reply:
done